src/plugins/intel_cpu/tests/functional/single_layer_tests/select.cpp

```diff
@@ -202,4 +202,104 @@ const auto noneCases = ::testing::Combine(::testing::ValuesIn(inShapesDynamicNon
 
 INSTANTIATE_TEST_SUITE_P(smoke_CompareWithRefsNone_dynamic, SelectLayerCPUTest, noneCases, SelectLayerCPUTest::getTestCaseName);
 
+
+class SelectLayerCPUStaticTest : public testing::WithParamInterface<selectParams>,
```

Comment:
Could you please elaborate on why `Select` op may not be tested within the existing test class?